Providers/SyncDisplayState.swift

```diff
@@ -0,0 +1,47 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Note: This is a rename of the `SyncStatusResolver.swift` file (which is deleted below). The logic in this file is what remained after all the obsolete sync code was removed. This rename seemed like a better representation of what's left.